examples/atari/atari_wrapper.py

```diff
@@ -23,7 +23,7 @@ def __init__(self, env, noop_max=30):
 
     def reset(self):
         self.env.reset()
-        noops = np.random.randint(self.noop_max) + 1
+        noops = self.unwrapped.np_random.randint(1, self.noop_max + 1)
```

Comment:
Leave some docs here to prevent further changes. Normally we would like to use ``np.random``.